homeassistant/components/google_assistant/trait.py

```diff
@@ -1047,3 +1036,67 @@ def query_attributes(self):
                             ATTR_ENTITY_ID: self.state.entity_id,
                             media_player.ATTR_INPUT_SOURCE: source
                         }, blocking=True, context=data.context)
+
+
+@register_trait
+class OpenCloseTrait(_Trait):
+    """Trait to open and close a cover.
+    https://developers.google.com/actions/smarthome/traits/openclose
+    """
+
+    name = TRAIT_OPENCLOSE
+    commands = [
+        COMMAND_OPENCLOSE
+    ]
+
+    @staticmethod
+    def supported(domain, features):
+        """Test if state is supported."""
+        return domain == cover.DOMAIN
+
+    def sync_attributes(self):
+        """Return opening direction."""
+        return {
+            'openDirection': ['UP']
```

Comment:
The 'openDirection' attribute is not required so we might not want to assume the 'Up' value until we plan on implementing that attribute in 'Cover'.

Reply:
This should be based on the device class. If it's not optional, let's leave it out for now.

Reply:
@giefca 
if i want to use up/down can instead of open/close
it it enough to add that line :             'openDirection': ['UP'] 
line 1050 ?
i prefer to speak with words up/down

Reply:
if your cover is closed when it is down you shouldn't have to add anything

Reply:
i am using the TCP command line cover, i dont know the state at all of my blinds
https://www.home-assistant.io/components/cover.command_line/
just wanted to ask , how to change the code, so i can say up/down
open/close isnt a good dutch translation for us, we dont say open close cover  , we say up/down for covers ... so up/down is in dutch (for covers) the way we open / close a cover :)